### PR TITLE
Routes and Navbar

### PR DESCRIPTION
**Pull request by Samah-Mustafa**

### CODE REVIEW

File: src/components/common/index.css [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+
+.navbar .nav-link:hover {
+  color: #b0faff !important;
+  font-family: "Myriad Pro", sans-serif;
+  font-weight: 400;
+  font-size: 17px;
+  letter-spacing: 1px;
+}
+
+.nav-link .special-color {
```
Comment (louisrli) [inline src/components/common/index.css:44]: for css classes, we should use camel case. so `.navLink .specialColor`. fix here and everywhere else

File: src/components/home/Home.jsx [hunk 2]
```diff
@@ -0,0 +1,13 @@
+import React from "react";
+import { Header } from "./sections/Header";
+
+export const Home = () => {
+  return (
+    <div>
+      <Header />
+      <br />
```
Comment (louisrli) [inline src/components/home/Home.jsx:8]: let's delete these 3 BRs here, generally we don't use BR like this. if you want to add space use a bottom margin on header

File: src/components/noMatch/NoMatch.jsx [hunk 3]
```diff
@@ -0,0 +1,5 @@
+import React from "react";
+
```
Comment (louisrli) [inline src/components/noMatch/NoMatch.jsx:2]: what does noMatch mean?

for directory names, let's use all lowercase
`components/nomatch`

File: src/components/signInUp/SignInUp.jsx [hunk 4]
```diff
@@ -0,0 +1,5 @@
+import React from "react";
```
Comment (louisrli) [inline src/components/signInUp/SignInUp.jsx:1]: `components/signinup/SignInUp.jsx`

**Review by louisrli — COMMENTED** [review]
generally looks good, please fix comments and make sure circleci config is in

**Review by basil-kawak — COMMENTED** [review]
@Samah-Mustafa I don't think we nee this branch anymore, let's delete it

**Pull request closed without merging**